### PR TITLE
Defined color code of application according to environment mode.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/utils/dataGridUtils.js [hunk 1]
```diff
@@ -95,3 +95,11 @@ export const chunkArray = (Array, chunkSize) => {
   }
   return results;
 };
+
+export const setAppColor = (mode) => {
+  for (const key in mode) {
+    if (mode.hasOwnProperty(key)) {
+      document.documentElement.style.setProperty(key, mode[key]);
```
Comment (pepcus-pratik) [inline src/utils/dataGridUtils.js]: What this the purpose of this code??

File: config/environmentVariable.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply config/environmentVariable.json]: Done.

File: src/actions/assetFilesActions.js [hunk 3]
```diff
@@ -26,3 +26,18 @@ export const fetchFileConfigFailedAction = errorMessage => ({
   type: 'FETCH_FILES_CONFIG_FAILED_ACTION',
   errorMessage,
 });
+
+export const loadedAppDataAction = () => ({
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/actions/assetFilesActions.js]: Done.

File: src/actions/assetFilesActions.js [hunk 4]
```diff
@@ -26,3 +26,18 @@ export const fetchFileConfigFailedAction = errorMessage => ({
...
+
+export const loadedAppDataSuccessAction = modeVariable => ({
+  type: 'LOADED_APP_DATA_SUCCESS_ACTION',
+  modeVariable,
+});
+
+export const loadedAppDataFailedAction = errorMessage => ({
+  type: 'LOADED_APP_DATA_FAILED_ACTION',
+  errorMessage,
+});
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/actions/assetFilesActions.js]: Done.

File: src/components/coreComponents/AppContainer.jsx [hunk 5]
```diff
@@ -1,12 +1,56 @@
...
+            </span>
+            {ERROR_MESSAGE_OF_LOAD_APP_DATA}
+          </div>
+        </div>
+      );
+    }
+    return null;
+  }
+}
+
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/components/coreComponents/AppContainer.jsx]: Done.

File: src/components/coreComponents/AppContainer.jsx [hunk 6]
```diff
@@ -1,12 +1,56 @@
...
-const AppContainer = () => (
-  <HashRouter>
-    <Route path={'/'} component={Routes} />
-  </HashRouter>
-);
+class AppContainer extends Component {
+  componentDidMount() {
+    this.props.loadedAppDataAction();
+    if (this.props.isAppLoaded) {
+      setAppColor(this.props.mode.environment === 'production' ? production : development);
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/components/coreComponents/AppContainer.jsx]: Done.

File: src/reducers/assetFilesReducer.js [hunk 7]
```diff
@@ -55,3 +81,9 @@ export const getFileData = state => state.assetFilesReducer.fileData;
 export const isLoading = state => state.assetFilesReducer.isLoading;
 
 export const getFilesConfig = state => state.assetFilesReducer.filesConfig;
+
+export const getModeVariable = state => state.assetFilesReducer.modeVariable;
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/reducers/assetFilesReducer.js]: Done.

File: src/reducers/assetFilesReducer.js [hunk 8]
```diff
@@ -55,3 +81,9 @@ export const getFileData = state => state.assetFilesReducer.fileData;
...
+
+export const getIsAppLoaded = state => state.assetFilesReducer.isAppLoaded;
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/reducers/assetFilesReducer.js]: Done.

File: src/sagas/assetFilesAPI.js [hunk 9]
```diff
@@ -22,3 +22,9 @@ export const fetchFileConfig = () => (
     url: 'files/filesConfig.json',
   })
 );
+
+export const loadAppData = () => (
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/sagas/assetFilesAPI.js]: Done.

File: src/sagas/assetFilesSaga.js [hunk 10]
```diff
@@ -54,3 +56,18 @@ export function* fetchFilesConfigSaga() {
     yield put(fetchFileConfigFailedAction(errorMessage));
   }
 }
+
+export function* loadAppDataSaga() {
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/sagas/assetFilesSaga.js]: Done.

File: src/utils/dataGridUtils.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/utils/dataGridUtils.js]: It is used to set color variables according to environment mode in CSS files.

File: src/config/productionColorCode.json [hunk 11]
```diff
@@ -0,0 +1,61 @@
+{
```
[the remark this answers is not in the document]
Comment (BhagyashreeBhoyar) [reply src/config/productionColorCode.json]: Done.

**Pull request merged**